template.tpl

```diff
@@ -1,12 +1,4 @@
-﻿___TERMS_OF_SERVICE___
```

Comment:
Let's keep that part in the template just in case Google introduces client template gallery at some point.